services/QuillLMS/app/assets/stylesheets/pages/premium_hub/account-management.scss

```diff
@@ -85,9 +85,19 @@
         background-color: $quill-green-1;
       }
       .data-table-row {
+        padding: 0px;
         &:hover {
           background-color: $quill-green-1 !important;
         }
+        td:first-of-type {
+          min-width: 570px !important;
```

Comment:
Maybe make a note here with a link to the line number that needs to get updated if this changes?

Reply:
added comments!